earth_enterprise/src/server/geserver

```diff
@@ -84,15 +86,61 @@ stop() {
   echo
   echo $dashes
   # only stop if already running...
-  if [ -f /var/opt/google/pgsql/data/postmaster.pid ] ; then
+  if [ -f $pidfile_postmaster ] ; then
     run_as_user $GEPGUSER "$gebin_root/pg_ctl -D /var/opt/google/pgsql/data -l /var/opt/google/pgsql/logs/pg.log stop";
-    rm -f /var/opt/google/pgsql/data/postmaster.pid
+    rm -f $pidfile_postmaster
   fi
   echo $dashes
   echo 'Done Stopping Google Earth Enterprise Server'
   echo
 }
 
+status_postmaster() {
+  if [ -s $pidfile_postmaster ] ; then
+    pid_postmaster=`head -1 $pidfile_postmaster`
+    # redirect to /dev/null because pg_ctl --silent option doesnt work for status subcommand
+    run_as_user $GEPGUSER "$gebin_root/pg_ctl -D /var/opt/google/pgsql/data status" >& /dev/null
+    returncode=$?
+  else
+    returncode=1
+  fi
+  if [ $returncode -eq 0 ]; then
+    echo "postmaster: OK"
+  else
+    echo "postmaster: Not running"
+  fi
+  return $returncode
+}
+
+status_gehttpd() {
```

Comment:
Minor: Reuse `check_daemon` from `earth_enterprise/src/fusion/autoingest/sysman/gefusion`.

We should, actually, share the code instead of rewriting the same function multiple times, but, since that may expand the scope of this ticket too much, we can use a re-implementation for now.

Reply:
Yes, just like src/installer/common.sh, the services would benefit from some common source.  I thought about it but came to the same conclusion.

Reply:
Yes, that could be a separate ticket.